src/imap-client.js

```diff
@@ -919,9 +921,15 @@
 
         if (options.answered === true) {
             add.push(ANSWERED_FLAG);
-        } else if (options.answered === true) {
+        } else if (options.answered === false) {
             remove.push(ANSWERED_FLAG);
         }
+        
+        if (options.answered === true) {
```

Comment:
Oops haha I forgot to update the if statement. The idea for flagging was because `expunge` is expensive and not needed if the goal is just to move the trash. Technically could just use move to trash but that would require the path of the trash, simply flagging it is much quicker and easier